token_server/README.md

```diff
@@ -1,4 +1,32 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+## Deploy to Google App Engine
```

Comment:
Can we add more of a header paragraph about who these instructions are for (maybe even date and person writing?) Mostly came to mind because I wouldn't want anyone who is earth engine whitelisted and just wants to access locally thinking they need this

Reply:
I don't think we expect anyone but developers to be viewing these READMEs, no? But added some text.

Reply:
Agreed only developers but developers like instructions too

Reply:
I was addressing your "earth engine whitelisted" scenario: I don't think we expect anyone except Meylakh and Katie to be reading these docs. Alex Nawar shouldn't be getting here. Are these docs ok?

Reply:
Added some comments!